Initial Stars documentation (#1929)
o3de/o3de.org: content/docs/user-guide/gems/reference/rendering/stars.md

---
linkTitle: Stars
title: Stars Gem

> description: The Stars Gem provides physically-based animated resolution-independant distant stars.

Reviewer: ```suggestion
description: The Stars Gem provides physically-based animated resolution-independent distant stars.
```

toc: true
---

> The **Stars** Gem provides physically-based animated resolution-independant distant stars.

Reviewer: ```suggestion
The **Stars** Gem provides physically-based animated resolution-independent distant stars.
```

> For more information on using the functionality provided by this gem see the [Stars Component](/docs/user-guide/components/reference/atom/stars.md) documentation.

Reviewer 2: ```suggestion
For more information on using the functionality provided by this gem refer to the [Stars Component](/docs/user-guide/components/reference/atom/stars.md) documentation.
```
